bin/01_train_classifier.py

```diff
@@ -5,13 +5,16 @@
 
 from traffic_cam import classifier, paths
 
+import os
```

Comment:
I suggest we import a build-in module earler.

Reply:
PyCharm > Optimize Imports agrees with you ;)